charts/portal/templates/deployment-backend-services.yaml

```diff
@@ -174,6 +174,10 @@ spec:
           value: "{{ .Values.backend.services.uploadServiceDocumentTypeIds.mediaTypeIds1.mediaTypeId1 }}"
         - name: "SERVICES__UPLOADSERVICEDOCUMENTTYPEIDS__1__MEDIATYPES__2"
           value: "{{ .Values.backend.services.uploadServiceDocumentTypeIds.mediaTypeIds1.mediaTypeId2 }}"
+        - name: "SERVICES__UPLOADSERVICEDOCUMENTTYPEIDS__2__DOCUMENTTYPEID"
+          value: "{{ .Values.backend.services.uploadServiceDocumentTypeIds.documentTypeId2 }}"
+          name: "SERVICES__UPLOADSERVICEDOCUMENTTYPEIDS__2__MEDIATYPES__0"
+          value: "{{ .Values.backend.services.uploadServiceDocumentTypeIds.mediaTypeIds0.mediaTypeId0 }}"
```

Comment:
Please adjust to `.Values.backend.services.uploadServiceDocumentTypeIds.mediaTypeIds2.mediaTypeId0`

Reply:
Request change done